echopop/survey.py

```diff
@@ -213,20 +216,299 @@ def stratified_analysis(
+            `n_lags`.
+        force_lag_zero: bool
+            See the `variogram_parameters` argument in
+            :fun:`echopop.spatial.variogram.empirical_variogram` for more details on
+            `force_lag_zero`.
+        max_range: Optional[float]
+            An optional input defining the maximum lag distance range that will be computed for
+            fitting the theoretical variogram parameters.
+        standardize_coordinates: bool
+            When set to `True`, transect coordinates are standardized using reference coordinates.
+        variable: Literal["biomass", "abundance"]
+            Transect data values used for fitting the variogram. This includes two options:
+            "abundance" and "biomass", with the default being "biomass". These inputs correspond
+            to fitting the empirical and theoretical variograms on "number density" and "biomass
+            density", respectively.
+        initial_values: Optional[Union[List[Tuple[str, float]], Dict[str, Dict[str, float]]]]
+            An optional list of tuples or a dictionary that includes labeled values that are used
+            for initial variogram model parameter optimization. This uses the following format:
+                - `initial_values = [("param1", initial_value1), ("param2", initial_value2)]`
+                - `initial_values = {("param1": initial_value, "param2": initial_value2)}`
+            Example: `initial_values = [("nugget", 0.0), ("decay_power", 1.5)]`
+        lower_bounds: Union[List[Tuple[str, float]], Dict[str, Dict[str, float]]]
+            A list of tuples or dictionary that includes labeled values that are used to set the
+            lower bounds (or minimum) of each parameter during optimization. This uses an identical
+            format as the `initial_values` argument.
+        upper_bounds: Union[List[Tuple[str, float]], Dict[str, Dict[str, float]]]
+            A list of tuples or dictionary that includes labeled values that are used to set the
+            upper bounds (or maximum) of each parameter during optimization. This uses an identical
+            format as the `initial_values` argument. When not configured, these values default to
+            infinity.
+        max_fun_evaluations: int
+            The maximum number of evaluations. Defaults to 500.
+        cost_fun_tolerance: float
+            Threshold used for determining convergence via incremental changes of the cost function.
+            Defaults to 1e-6.
+        solution_tolerance: float
+            Threshold used for determining convergence via change of the independent variables.
+            Defaults to 1e-8.
+        gradient_tolerance: float
+            Threshold used for determining convergence via the gradient norma. Defaults to 1e-8.
+        finite_step_size: float
+            The relative step sizes used for approximating the Jacobian via finite differences.
+        trust_region_solver: Literal["exact", "float"]
```

Comment:
Is the suggested change below right?
```suggestion
        trust_region_solver: Literal["exact", "base"]
```